.github/workflows/release.yml

```diff
@@ -0,0 +1,393 @@
+          APP_VERSION=${GITHUB_REF/refs\/tags\/v/}
+          echo "Tag: $GITHUB_REF Version = $APP_VERSION"
+          echo "::set-output name=app_version::$APP_VERSION"          
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: true
+          prerelease: false
+
+  linux-jpackage:
+    runs-on: ubuntu-18.04
+    needs: create-release
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Gluon's GraalVM
+        uses: gluonhq/setup-graalvm@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install libraries
+        run: |
+          sudo apt update
+          sudo apt install libasound2-dev libavcodec-dev libavformat-dev libavutil-dev libgl-dev libgtk-3-dev libpango1.0-dev libxtst-dev
+
+      - name: Make staging directory
+        run: |
+          mkdir staging
+
+      - name: Gluon License
+        uses: gluonhq/gluon-build-license@master
+        with:
+          gluon-license: ${{ secrets.GLUON_LICENSE }}
+          dir: App
+
+      - name: Setup java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Build and package
+        run: mvn clean install && mvn package -f App
+
+      - name: Create Bundle using jpackage
+        id: outputfile
+        run: |
+          .github/scripts/build_app_linux.sh \
+          --type deb
+          mv App/target/installer/*.deb staging/ChatApp-${{ env.APP_VERSION }}.deb
+          echo ::set-output name=pathDeb::staging/ChatApp-${{ env.APP_VERSION }}.deb
+          .github/scripts/build_app_linux.sh \
+          --type rpm
+          mv App/target/installer/*.rpm staging/ChatApp-${{ env.APP_VERSION }}.rpm
+          echo ::set-output name=pathRpm::staging/ChatApp-${{ env.APP_VERSION }}.rpm
+        env:
+          PROJECT_VERSION: 1.1.0-SNAPSHOT
+          APP_VERSION: ${{needs.create-release.outputs.app_version}}
+
+      - name: Upload installer (deb)
+        uses: actions/upload-artifact@v2
+        with:
+          name: InstallerPackageLinux (deb)
+          path: ${{ steps.outputfile.outputs.pathDeb }}
+
+      - name: Upload installer (rpm)
+        uses: actions/upload-artifact@v2
+        with:
+          name: InstallerPackageLinux (rpm)
+          path: ${{ steps.outputfile.outputs.pathRpm }}
+
+      - name: Upload Release Asset (deb)
+        id: upload-release-asset-deb
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{needs.create-release.outputs.upload_url}}
+          asset_path: ${{ steps.outputfile.outputs.pathDeb }}
+          asset_name: ChatApp-${{needs.create-release.outputs.app_version}}.deb
```

Comment:
Should we at least add the platform name to the filename to be able to more quickly identify what artifact a user should have to download? For instance, for linux: `ChatApp-1.2-linux.deb`.